app/controllers/api/v1/customers/finder_controller.rb

```diff
@@ -7,7 +7,7 @@ def show
 
   def index
     finder = params.keys[0]
-    render json: Customer.where(finder => params[finder])
+    render json: Customer.where("lower(#{finder}) = ?", params[finder].downcase)
```

Comment:
@meyerhoferc I would stay away from string interpolation in your SQL (potential security vulnerabilities), and I don't believe that you need that (per our discussion this morning about not worrying about the finder itself being case insensitive).

Re `.downcase`, is this being stored in the DB in lower case? It seems like there's a potential loss there in terms of displaying the information later. See if you can find any info on `citext` as a Postgres data type in Rails.

One other note: the argument to your `.where` call, you can use strong params to limit what you're getting in from your params hash, which could potentially simplify this a little bit. The one trick is that the params aren't nested, so you'll need to adjust your normal strong params slightly.